buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -224,6 +224,7 @@ def build(OPENJDK_REPO, OPENJDK_BRANCH, SHAS, OPENJ9_REPO, OPENJ9_BRANCH, OMR_RE
                     string(name: 'SLACK_CHANNEL', value: SLACK_HANDLE),
                     string(name: 'RESTART_TIMEOUT', value: RESTART_TIMEOUT),
                     string(name: 'RESTART_TIMEOUT_UNITS', value: RESTART_TIMEOUT_UNITS)]
```

Comment:
This is not the last element of the array, remove `]` add `,`:
`string(name: 'RESTART_TIMEOUT_UNITS', value: RESTART_TIMEOUT_UNITS),`

Reply:
Good catch, likely a result of a merge conflict. Fixed.